commit f24297abb3042ce3721abb817590d2b8964662a7
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Wed Oct 31 14:57:56 2018 +0000

    media: adv7180: Add YPrPb support for ADV7282M
    
    The ADV7282M can support YPbPr on AIN1-3, but this was
    not selectable from the driver. Add it to the list of
    supported input modes.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/media/i2c/adv7180.c b/drivers/media/i2c/adv7180.c
index 931c0408b046..e0584040eca8 100644
--- a/drivers/media/i2c/adv7180.c
+++ b/drivers/media/i2c/adv7180.c
@@ -1229,6 +1229,7 @@ static const struct adv7180_chip_info adv7282_m_info = {
 		BIT(ADV7182_INPUT_SVIDEO_AIN1_AIN2) |
 		BIT(ADV7182_INPUT_SVIDEO_AIN3_AIN4) |
 		BIT(ADV7182_INPUT_SVIDEO_AIN7_AIN8) |
+		BIT(ADV7182_INPUT_YPRPB_AIN1_AIN2_AIN3) |
 		BIT(ADV7182_INPUT_DIFF_CVBS_AIN1_AIN2) |
 		BIT(ADV7182_INPUT_DIFF_CVBS_AIN3_AIN4) |
 		BIT(ADV7182_INPUT_DIFF_CVBS_AIN7_AIN8),
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1665_f24297abb3042ce3721abb817590d2b8964662a7.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1665_f24297abb3042ce3721abb817590d2b8964662a7.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
